src/unzip/seekable_http_reader.rs

```diff
@@ -564,23 +564,33 @@ pub(crate) struct SeekableHttpReader {
 
 impl Seek for SeekableHttpReader {
     fn seek(&mut self, pos: std::io::SeekFrom) -> std::io::Result<u64> {
-        // TODO used checked arithmetic when stabilized
```

Comment:
This TODO actually was in reference to a proposal to allow checked arithmetic between disparate types, e.g. `u64` and `i64`. However using `try_into` is just as good.